apps/redis/templates/app/dc.yml.j2

```diff
@@ -25,6 +25,22 @@ spec:
       containers:
         - name: redis
           image: "{{ redis_app_image_name }}:{{ redis_app_image_tag }}"
+          livenessProbe:
+            exec:
+              command:
+                - "/bin/bash"
+                - "-c"
+                - "redis-cli info"
```

Comment:
You can use the `ping` command: https://redis.io/commands/ping

I think this is more accurate with what we are looking for.